src/core/ext/lb_policy/round_robin/round_robin.c

```diff
@@ -76,15 +77,35 @@ int grpc_lb_round_robin_trace = 0;
  * Once a pick is available, \a target is updated and \a on_complete called. */
 typedef struct pending_pick {
   struct pending_pick *next;
+
+  /* polling entity for the pick()'s async notification */
   grpc_polling_entity *pollent;
+
+  /* the initial metadata for the pick. See grpc_lb_policy_pick() */
+  grpc_metadata_batch *initial_metadata;
```

Comment:
It doesn't look like this is used anywhere.  I suspect it's not needed here anymore.